src/main/io/serial_msp.c

```diff
@@ -1804,19 +1801,13 @@ static bool processInCommand(void)
         currentProfile->pidProfile.dterm_lpf_hz = read16();
         currentProfile->pidProfile.yaw_lpf_hz = read16();
         break;
-    case MSP_SET_ADVANCED_TUNING:
+    case MSP_SET_PID_ADVANCED:
         currentProfile->pidProfile.rollPitchItermIgnoreRate = read16();
         currentProfile->pidProfile.yawItermIgnoreRate = read16();
         currentProfile->pidProfile.yaw_p_limit = read16();
         currentProfile->pidProfile.deltaMethod = read8();
         masterConfig.batteryConfig.vbatPidCompensation = read8();
         break;
-    case MSP_SET_SPECIAL_PARAMETERS:
-        currentControlRateProfile->rcYawRate8 = read8();
-        masterConfig.rxConfig.airModeActivateThreshold = read16();
```

Comment:
@mikeller I rather want to keep this PR simple. The ordering of parameters would be different as the 3.0 has rc_smoothing in between. The airmode threshold isnt that crucial parameter at all so I am going to leave it as it is.